app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
```

Comment:
- [ ] You have done a great job implementing relationships, considering that using user as author is very confusing. The only thing you missed here, is adding foreign key to user like: 
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Without foreign key you cannot connect user with posts, you will be getting an error: 
<img width="1158" alt="Screenshot 2022-08-11 at 20 14 38" src="https://user-images.githubusercontent.com/77038610/184180436-81d9736c-aaaa-49b4-b432-1413f17f2ada.png">

after adding foreign keys everything works as expected 👍🏻

Reply:
This change has been done well